src/styles/blocks/cards.scss

```
@@ -0,0 +1,119 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  flex-direction: column;
+
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $border-color-card;
+  border-radius: 5px;
+
+  transition: 0.3s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```